internal/pkg/security/seccomp/seccomp_supported.go

```diff
@@ -45,7 +45,7 @@ var scmpArchMap = map[specs.Arch]lseccomp.ScmpArch{
 }
 
 var scmpActionMap = map[specs.LinuxSeccompAction]lseccomp.ScmpAction{
-	specs.ActKill:  lseccomp.ActKill,
+	specs.ActKill:  lseccomp.ActKillThread,
```

Comment:
Hmm why ?

Reply:
Oh, otherwise I was getting a linting failure telling me to make this change because ActKill was deprecated.   I did hit it once in another context too.  I don't know why it isn't showing up on CI checks.  Probably this should be in a separate commit, maybe even a separate PR but I don't know that it's worth that.

Reply:
Ok, yes that's fine in this PR